cdap-ui/app/styles/fonts.less

```diff
@@ -55,93 +55,166 @@
+  margin-left: -1em;
+  color: rgb(0, 0, 0);
+}
+.icon-delete:before {
+  content: "\e900";
+}
+.icon-start:before {
+  content: "\e901";
+}
+.icon-clone:before {
+  content: "\e902";
+}
+.icon-calendar:before {
+  content: "\e903";
+}
+.icon-zoomin:before {
+  content: "\e904";
+}
+.icon-zoomout:before {
+  content: "\e905";
+}
+.icon-clean:before {
+  content: "\e906";
+}
+.icon-fit:before {
+  content: "\e907";
+}
+.icon-validate:before {
+  content: "\e908";
+}
+.icon-savedraft:before {
+  content: "\e909";
+}
+.icon-publish:before {
+  content: "\e90a";
+}
+.icon-import:before {
+  content: "\e90b";
+}
+.icon-export:before {
+  content: "\e90c";
+}
+.icon-draft:before {
+  content: "\e61e";
+}
+.icon-stopped:before {
+  content: "\e61f";
+}
+.icon-paused:before {
+  content: "\e620";
+}
+.icon-failed:before {
+  content: "\e621";
+}
+.icon-running:before {
+  content: "\e622";
+}
 .icon-hydrator:before {
-	content: "\e61d";
+  content: "\e61d";
 }
 .icon-SchemaEdge:before {
-	content: "\e61c";
+  content: "\e61c";
 }
 .icon-ETLBatch:before {
-	content: "\e61a";
+  content: "\e61a";
 }
 .icon-ETLRealtime:before {
-	content: "\e61b";
+  content: "\e61b";
 }
 .icon-adapter:before {
-	content: "\e600";
+  content: "\e600";
 }
 .icon-app:before {
-	content: "\e601";
+  content: "\e601";
 }
 .icon-applicationtempaltes:before {
-	content: "\e602";
+  content: "\e602";
 }
 .icon-avro:before {
-	content: "\e603";
+  content: "\e603";
 }
 .icon-configuration:before {
-	content: "\e604";
+  content: "\e604";
 }
 .icon-datasets:before {
-	content: "\e605";
+  content: "\e605";
 }
 .icon-datatypes:before {
-	content: "\e606";
+  content: "\e606";
 }
 .icon-ETLsinks:before {
-	content: "\e607";
+  content: "\e607";
 }
 .icon-ETLsources:before {
-	content: "\e608";
+  content: "\e608";
 }
 .icon-ETLtemplates:before {
-	content: "\e609";
+  content: "\e609";
 }
 .icon-ETLtransforms:before {
-	content: "\e60a";
+  content: "\e60a";
 }
 .icon-fist:before {
-	content: "\e60b";
+  content: "\e60b";
 }
 .icon-jms:before {
-	content: "\e60c";
+  content: "\e60c";
 }
 .icon-kafka:before {
-	content: "\e60d";
+  content: "\e60d";
 }
 .icon-mapreduce:before {
-	content: "\e60e";
+  content: "\e60e";
 }
 .icon-plugin-stream:before {
-	content: "\e60f";
+  content: "\e60f";
 }
 .icon-programtypes:before {
-	content: "\e610";
+  content: "\e610";
 }
 .icon-projection:before {
-	content: "\e611";
+  content: "\e611";
 }
 .icon-reset:before {
-	content: "\e612";
+  content: "\e612";
 }
 .icon-service:before {
-	content: "\e613";
+  content: "\e613";
 }
 .icon-spark:before {
-	content: "\e614";
+  content: "\e614";
 }
 .icon-streams:before {
-	content: "\e615";
+  content: "\e615";
 }
 .icon-systempreferences:before {
-	content: "\e616";
+  content: "\e616";
 }
 .icon-systemsecruity:before {
-	content: "\e617";
+  content: "\e617";
 }
 .icon-tigon:before {
-	content: "\e618";
+  content: "\e618";
 }
-.icon-workflow:before {
-	content: "\e619";
+.icon-worlflow:before {
```

Comment:
change this to workflow. Might need to inform Sean, so that we don't have to do this every time we do an icon update.